superset-frontend/src/dashboard/App.jsx

```diff
@@ -25,22 +25,28 @@ import { HTML5Backend } from 'react-dnd-html5-backend';
 import { DynamicPluginProvider } from 'src/components/DynamicPlugins';
 import setupApp from '../setup/setupApp';
 import setupPlugins from '../setup/setupPlugins';
-import DashboardContainer from './containers/Dashboard';
+import DashboardRoute from './components/DashboardPage';
```

Comment:
```suggestion
import DashboardPage from './components/DashboardPage';
```

Let's have more consistent component name? `DashboardPage.tsx` probably also don't need to live under `./components`.

Reply:
Oh good catch, that's an auto-refactor artifact. I will also move `DashboardPage` to /containers.